airbyte-integrations/connectors/destination-jdbc/src/main/java/io/airbyte/integrations/destination/jdbc/constants/GlobalDataSizeConstants.java

```diff
@@ -0,0 +1,12 @@
+package io.airbyte.integrations.destination.jdbc.constants;
+
+import io.aesy.datasize.ByteUnit.IEC;
+import io.aesy.datasize.DataSize;
+
+public interface GlobalDataSizeConstants {
+  // 256 MB to BYTES as comparison will be done in BYTES
+  int MAX_BATCH_SIZE_BYTES = DataSize.of(256L, IEC.MEBIBYTE).toUnit(IEC.BYTE).getValue().intValue();
+  // This constant determines the max size of uploaded chunk of file(e.g. 15 GB / 256 megabytes ≈ 58 MB per chunk of file)
+  // see StagingFilenameGenerator.java:28
+  long MAX_BYTE_PARTS_PER_FILE = DataSize.of(15L, IEC.GIBIBYTE).toUnit(IEC.BYTE).getValue().longValue();
```

Comment:
why 15gb?

Reply:
It is just constant we assume the biggest file could be of 15GB of data. After, division this size on 256 MB and after have a chunks ≈ 58 MB each. Maybe, it is possible to find more efficient algorithm.